modules/pinger/src/test/java/org/hawkular/component/pinger/PingerTest.java

```diff
@@ -75,39 +75,11 @@ public void testPostPinger() throws Exception {
     public void testSslPinger() throws Exception {
 
         Pinger pinger = new Pinger();
-        PingDestination destination = new PingDestination("123","https://www.perfcake.org");
+        PingDestination destination = newDestination("https://www.perfcake.org", "GET");
```

Comment:
Can you add a server making use of SNI as well? Example of a server that makes use of SNI: https://mail.kroehling.de . But don't add my address to the test, as this machine is in an unstable environment ,so, your test might fail without reason :)